src/Pages/ProductList/Components/Aside/Aside.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import 'Pages/ProductList/Components/Aside/Aside.scss';
```

Comment:
```suggestion
import './Aside.scss';
```
Depth 가 깊어지면 유지보수가 어려울 수 있습니다.
src 파일에 접근할 때에는 상대경로로 설정을 해 주세요:)
그리고, 위 방식으로 하는게 연관성을 쉽게 알 수 있습니다!

Reply:
상대경로 수정완료하였습니다